### DISCUSSION TITLE
add gitlens.togglelineblame to vscode menu

### DISCUSSION THREAD

**Discussion by think120** [gitkraken/vscode-gitlens#1700] [Q&A] [answered]
Could I please ask whether it is possible to add `gitlens.togglelineblame` to vscode menu? Many thanks!

**Comment by eamodio** [comment]
Can you please provide more details as to which menu you'd like to have it on?

**Reply by think120** [reply]
Hi @eamodio ,  I want to add the following option to the right click menu, so I can toggle line blame on/off directly from right click menu. Many thanks!

![image](https://user-images.githubusercontent.com/78618719/140039353-8e9c32e0-cd3f-40c2-b809-fa1400eb7129.png)

**Comment by eamodio** [comment, marked as answer]
You can do this today by toggling this setting via the GitLens Settings editor (_GitLens: Open Settings_ from the command palette):

![image](https://user-images.githubusercontent.com/641685/140235437-0a9ac853-1719-426f-a5a9-a731272b8b55.png)

Hope that helps!
